nuget/lib/dependabot/nuget/update_checker/repository_finder.rb

```diff
@@ -59,7 +59,7 @@ def build_url_for_details(repo_details)
           end
           if search_url
             details[:search_url] =
-              search_url + "?q=#{dependency.name.downcase}&prerelease=true"
+              search_url + "?q=#{dependency.name.downcase}&prerelease=true&semVerLevel=2.0.0"
```

Comment:
Would this still include semver level 1 releases?

Reply:
Yes it does